### ISSUE TITLE
Migrate CI

### ISSUE THREAD

**Issue by bmaupin** [bmaupin/devops-cheatsheets#1]
~~The migration is already done, so it should just be a matter of updating README.md à la https://github.com/bmaupin/go-epub/commit/8dc2323be8d39b2322f4cc7a95adc90f69ce318a~~

Hmm, it seems Travis CI may no longer offer free builds for open-source projects: https://news.ycombinator.com/item?id=25338983

It might be better to migrate to GitHub Actions...

It seems Chrome may already be installed in the default ubuntu image (https://github.community/t/how-to-install-google-chrome-browser-in-github-actions-workflow/18471/2)

So maybe just start with a blank image: https://github.com/actions/starter-workflows/blob/main/ci/blank.yml

And try to run build.sh? e.g.

```yaml
# This is a basic workflow to help you get started with Actions

name: CI

# Controls when the action will run. 
on:
  # Triggers the workflow on push or pull request events but only for the $default-branch branch
  push:
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - uses: actions/checkout@v2

      - name: Build
        # or ./build.sh?
        run: build.sh
```

https://remarkablemark.org/blog/2020/12/12/headless-test-in-github-actions-workflow/

**Comment by bmaupin** [comment]
Hmm, it seems Travis CI may no longer offer free builds for open-source projects: https://news.ycombinator.com/item?id=25338983

It might be better to migrate to GitHub Actions...
